scripts/utils.sh

```diff
@@ -65,6 +65,12 @@ function setDockerComposeCLIOptions () {
         export dockerComposeFilesCLIOptions="$dockerComposeFilesCLIOptions -f ../$file"
     done
 
+    # Add restore file if restore env is set
+
+    if [ "$BACKUP_RESTORE" == "true" ]; then
```

Comment:
This is confusing. Please rename to something clearer.
Eg, `RESTORE`, `RESTIC_RESTORE`, `RESTORE_WITH_RESTIC`...